spec/rails_helper.rb

```diff
@@ -90,10 +90,6 @@
   config.include ViewComponent::TestHelpers, type: :component
   config.include Rails.application.routes.url_helpers
 
-  config.before :each do
-    clear_enqueued_jobs
-  end
-
```

Comment:
Why not?

Reply:
Unnecessary - the way ActiveJob works in test is that it creates a brand new TestQueue adapter before each test with a blank `enqueued jobs` list. No point in clearing it.